frontend/scss/_mixins.scss

```diff
@@ -1,3 +1,5 @@
+@import 'variables';
```

Comment:
I think we are using the fully qualified `@import '_variables.scss'` in most cases